k8s.jsonnet

```diff
@@ -16,6 +17,20 @@
     // you're actually running, and must manually re-pull to get an updated copy.
     image_tag: "latest",
 
+    // For each component, whether to deploy that component.
+    enabled:: {
```

Comment:
style nitpick: the `::` isn't needed here.

In jsonnet, `key::` instead of `key:` denotes a "hidden" field. Hidden fields do not render in the output and do not show up when you do `std.objectKeys()` (there is an alternate method to get all keys including hidden ones).

Generally the idea is that you can use a hidden key for "private" fields or intermediate values you don't want to include in the output.

In our `docker-compose.jsonnet`, the config options are all at the top level. This requires them all to be hidden keys to avoid rendering in the final output. IMO this was a mistake, just one we haven't bothered to correct, which is why I opted for a hidden top-level `config` here. Since `config` is hidden, it won't be rendered at all so the keys inside it do not need to also be hidden. It does no harm but is redundant.